pkg/model/run.go

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright © 2019 One Concern
+ *
+ */
+
+package model
+
+import (
+	"time"
+)
+
+const (
+	Version = 1
+)
+
+type Category struct {
+	Name        string      `json:"name" yaml:"name"`               // Unique name for the category
+	Message     string      `json:"message" yaml:"message"`         // Message describing the category.
+	Contributor Contributor `json:"contributor" yaml:"contributor"` // Credentials for who created the category.
+	Version     int         `json:"version" yaml:"version"`         // Version of the run Schema.
+}
+
+type Run struct {
+	Name           string      `json:"name" yaml:"name"`                     // Name for the run
+	InstanceID     string      `json:"instanceid" yaml:"instanceid"`         // External ID used for a run. Not guaranteed to be unique.
```

Comment:
This is set by the pipeline?

Reply:
Pipeline may have a notion of a unique ID that they would like to preserve.